### ISSUE TITLE
New version 0.3.1 breaks energy.

### ISSUE THREAD

**Issue by tonioqiu** [AllTheMods/ATM-4#117]
After updating to 0.3.1, all energy sistemas shuts down, cant even put lava into the lava generator.

**Comment by Asshall** [comment]
Did you fix it ? Did you try to give yourself a new machine ? If it is part of a mod that have been updated or got it's config changed you might need to get a new one.

**Comment by tonioqiu** [comment]
No, I havent fix it. Sometimes it works and sometimes it doesn't. I have a server set up and running. It works all fine at first, but after certain server uptime, the power system stops working. The solar panels wont generate when in direct sunlight, the lava generator consumes lava but doesnt produce lava
